### DISCUSSION TITLE
Welcome to laravel-vite-docker Discussions!

### DISCUSSION THREAD

**Discussion by elvinlari** [elvinlari/laravel-vite-docker#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kapilpundit** [comment]
Hi @elvinlari ,
I used your project setup found on [Medium](https://betterprogramming.pub/dockerize-laravel-vite-react-application-in-your-development-environment-a118aea4a02d), and its a wonderful article. However, I am stuck at one thing. I am not able to call any laravel routes.
The application is running at http://localhost:8000. But when I add some more routed in the routes/web.php, they show me a 404.

I am using this command to run the application:
docker-compose up -d nginx
docker-compose run --rm --service-ports npm run dev

Is the '--service-ports' option causing this, if yes then what should be done here.
Thanks,

**Reply by kapilpundit** [reply]
I figured it out. It was my rotes cache that was not updated. just executed `artisan route:clear` and the new routes were accessible.
